ibm/service/power/resource_ibm_pi_image.go

```diff
@@ -188,8 +188,20 @@ func ResourceIBMPIImage() *schema.Resource {
 					},
 				},
 			},
+			Arg_UserTags: {
+				Description: "The user tags attached to this resource.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				ForceNew:    true,
```

Comment:
User tags cannot be updated. Why is this incorrect? We don't want users to try to update the fields thinking it will do something?

Reply:
@hkantare Since are only supporting this at creation, `forceNew` is okay here. The management of the `userTags` will be handle by **Global Tagging** at this moment. Like @ismirlia said we won't be allowing the user to update this.

Reply:
@michaelkad Why can't we adapt the same approach like other services team
During create/update always use Global Tagging API's.
This gives capability for user's to add /update tags

Reply:
@hkantare We will make a change to let the user update the tags using the Global Tagging API's.